app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

```diff
@@ -12,6 +12,10 @@ navigation: 20.40
 navigation_title: If you want to do a teaching apprenticeship
 navigation_description: A teacher degree apprenticeship (TDA) lets you work at a school and earn a salary while getting a degree and qualified teacher status (QTS).
 talk_to_us: false
+expander:    
```

Comment:
@emmaachesongray We should remove this from the front matter given we are not using the expander on this page at the moment